src/OpenLoco/src/Objects/LandObject.cpp

```diff
@@ -10,6 +10,14 @@
 
 namespace OpenLoco
 {
+    // Zoom levels are mandatory with DAT. Consider changing this with new object format.
+    constexpr auto const kNumTerrainImages = 19;
```

Comment:
What i have been doing for objects that don't have too much dynamic is listing out every single image offset and naming them. That allows for @LeftofZen to then take the names and apply them to the object editor. These 19*3 (kTerrainFlatImageOffset) image offsets we could name them i think with ease and the (kNumImagesPerGrowthStagePlusZoom) can be done in a follow on PR. (For an example see RoadObj::ImageIds)